lib/themes/dosomething/paraneue_dosomething/js/components/CampaignSlide.js

```diff
@@ -35,20 +41,22 @@ class CampaignSlide extends React.Component {
           <div className="wrapper">
             {this.renderHeader()}
 
-            {this.campaigns.map(campaign => {
-              // Fix for local files / data missing by preventing null error
-              const image_uri = campaign.cover_image.default != null ? campaign.cover_image.default.sizes.landscape.uri : '';
+            <div className="gallery -triad">
+              {this.state.campaigns.map(campaign => {
+                // Fix for local files / data missing by preventing null error
+                const image_uri = campaign.cover_image.default != null ? campaign.cover_image.default.sizes.landscape.uri : '';
 
-              const props = {
-                id: campaign.id,
-                image_uri: image_uri,
-                tagline: campaign.tagline,
-                title: campaign.title,
-                signup: this.controller.signup.bind(this.controller)
-              }
+                const props = {
+                  id: campaign.id,
+                  image_uri: image_uri,
+                  tagline: campaign.tagline,
+                  title: campaign.title,
+                  signup: this.controller.signup.bind(this.controller)
+                }
 
-              return <CampaignSignupCard {...props} key={campaign.id} />
-            })}
+                return <CampaignSignupCard {...props} key={campaign.id} />
```

Comment:
I'd also recommend always binding the methods in the constructor of their class. It removes some extra cruft from  references elsewhere, avoids a likely place for bugs if we happen to forget, and has a nice little micro-optimization in performance (only binds once, rather than re-binding on every single render). For example:

```js
class CampaignController {
  constructor() {
    this.signup = this.signup.bind(this);
  }

  signup() {
    // ... `this` will be CampaignController no matter what now
  }
}
```